html/captive-portal/lib/captiveportal/PacketFence/Controller/CaptivePortal.pm

```diff
@@ -136,9 +136,13 @@ sub checkForViolation : Private {
         # FIXME: there is not enough validation below
         my $vid      = $violation->{'vid'};
         my $SCAN_VID = 1200001;
+        my $POST_SCAN_VID = 1200004;
+        my $PRE_SCAN_VID = 1200005;
```

Comment:
These should really be moved out of this Controller. They belong in pf/constants/...